src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/IDocumentSnapshotExtensions.cs

```diff
@@ -27,12 +27,6 @@ internal static class IDocumentSnapshotExtensions
 
         var project = documentSnapshot.Project;
 
-        // If the document is an import document, then it can't be a component
-        if (project.IsImportDocument(documentSnapshot))
```

Comment:
This is mentioned in the commit message, but if you're looking at the PR as a whole let me explain here: This was the only call to `IsImportDocument`, and it was impossible for it to ever return `true` because 10 lines above this there is a check for `FileKind`. Import documents have their own file kind, so would never get this far.